packages/dotnet-sdk/src/TeamsFx/Conversation/NotificationOptions.cs

```diff
@@ -13,6 +13,30 @@ public class NotificationOptions
         /// </summary>
         public string BotAppId { get; set; } = string.Empty;
 
+        /// <summary>
```

Comment:
nit: Here we  use `notification connections`  to describe the `Store` member, while in the `IConversationReferenceStore` interface we use `notification target references`, it would be better to consistent the terminology in our SDK.

Reply:
Updated.